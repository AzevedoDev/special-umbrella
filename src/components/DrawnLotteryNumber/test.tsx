import { screen } from '@testing-library/react'
import { renderWithTheme } from 'utils/tests/helpers'

import LotterySortedNumber from '.'

describe('<LotterySortedNumber />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(
      <LotterySortedNumber lotteryNumber="01" />
    )

    expect(screen.getByText(/01/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
