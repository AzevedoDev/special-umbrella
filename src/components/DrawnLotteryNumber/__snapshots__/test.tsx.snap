// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LotterySortedNumber /> should render the heading 1`] = `
.c0 {
  background-color: #FFFFFF;
  width: 7.6rem;
  height: 7.6rem;
  border-radius: 50%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 1rem;
  font-family: 'Montserrat',-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Ubuntu,Cantarell,'Open Sans','Helvetica Neue',sans-serif;
}

.c1 {
  color: #333333;
  font-size: 2rem;
  font-weight: 700;
}

<main
  class="c0"
>
  <p
    class="c1"
  >
    01
  </p>
</main>
`
